.github/actions/import-db/action.yml

```diff
@@ -49,17 +53,21 @@ runs:
           echo "====="
           docker logs $DB5NAME || true
           echo "====="
+          # copy assetstore
+          echo Preparing assetstore
+          docker cp ${{ inputs.ASSETSTORE }} dspace${{ inputs.INSTANCE }}:/dspace/
+          echo "====="
```

Comment:
shouldn't here be a condition? What if ASSETSTORE variable is empty? Since it is not required, that can happen...